ratpack-pac4j/src/test/groovy/ratpack/pac4j/openid/OpenIdRpSpec.groovy

```diff
@@ -46,6 +48,17 @@ class OpenIdRpSpec extends RatpackGroovyDslSpec {
     }
 
     handlers {
+      prefix 'prefix', {
+        configurePac4j delegate
+      }
+
+      configurePac4j delegate
+    }
+
+  }
+
+  GroovyChain configurePac4j(GroovyChain chain) {
```

Comment:
I think we should we use a `GroovyChainAction` implementation here instead of this construct, it will be more idiomatic. I would also add the `noauth` handler only once, in the root of the test application.